eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Delegs.hs

```diff
@@ -70,7 +82,33 @@ data ConwayDelegsPredFailure era
       !(Map (RewardAcnt (EraCrypto era)) Coin)
   | -- | CERT rule subtransition Failures
     CertFailure !(PredicateFailure (EraRule "CERT" era))
-  deriving (Generic)
+  deriving (Generic, Typeable)
```

Comment:
Every type in Haskell Haskell has `Typeable` instance, it does not need to be derived
```suggestion
  deriving Generic
```